samples/subsys/display/lvgl/prj.conf

```diff
@@ -12,4 +13,5 @@ CONFIG_LV_USE_LOG=y
 CONFIG_LV_USE_LABEL=y
 CONFIG_LV_USE_BTN=y
 CONFIG_LV_USE_IMG=y
+CONFIG_LV_USE_MONKEY=y
```

Comment:
How does it affect flash memory usage?

Reply:
The `monkey` is an opaque structure and is allocated within the memory pool the zephyr manages for lvgl.
So the only overhead is actual code size for that feature (and the shell commands).

Built the sample for `mimxrt1060_evk`.
Without `CONFIG_LV_USE_MONKEY`:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      255856 B         8 MB      3.05%
             RAM:     1077224 B        32 MB      3.21%
            ITCM:          0 GB       128 KB      0.00%
            DTCM:          0 GB       128 KB      0.00%
           OCRAM:          0 GB       768 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
```

With `CONFIG_LV_USE_MONKEY=y`:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      257720 B         8 MB      3.07%
             RAM:     1077224 B        32 MB      3.21%
            ITCM:          0 GB       128 KB      0.00%
            DTCM:          0 GB       128 KB      0.00%
           OCRAM:          0 GB       768 KB      0.00%
        IDT_LIST:          0 GB         2 KB      0.00%
```
`257720-255856=1864`, so roughly `1KiB`.